src/__tests__/e2e.test.js

```diff
@@ -711,9 +711,7 @@ describe('e2e', () => {
   });
 
   it('root context items are accessible to children', () => {
-    const {
-      ReactTrackingContext,
-    } = require('../withTrackingComponentDecorator'); // eslint-disable-line global-require
+    const { ReactTrackingContext } = require('../useTracking'); // eslint-disable-line global-require
```

Comment:
This changed because I moved the context export into the `useTracking` file so that there wouldn't be a circular reference with the decorator importing and calling the hook and the hook importing the context object.